semgrep/tests/unit/test_version.py

```diff
@@ -0,0 +1,17 @@
+import time
+
+from semgrep.version import is_running_latest
+
+
+def test_version_check_caching(tmp_path):
+    tmp_cache_path = tmp_path / "semgrep_version"
+
+    time1 = time.time()
+    is_running_latest(tmp_cache_path)
+    time1 = time.time() - time1
+
+    time2 = time.time()
+    is_running_latest(tmp_cache_path)
```

Comment:
If you want a more accurate test you can actually assert that _fetch_latest_version was only called once

Reply:
Thoughts on copying the tests from https://github.com/returntocorp/bento/blob/master/tests/test_version.py ?